.travis.yml

```diff
@@ -1,35 +1,28 @@
 ---
-
-language: python
-python: "2.7"
 sudo: required
-dist: trusty
+language: python
+env:
+  - MOLECULE_DISTRO=fedora33   ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+  - MOLECULE_DISTRO=debian9    ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+  - MOLECULE_DISTRO=debian10   ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+#  - MOLECULE_DISTRO=ubuntu1604 ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
```

Comment:
Do we still intend to support `ubuntu1604` / `centos8` ?

Reply:
Absolutely. I think we can refactor the other pull request for centos8 after we merge this into master.

Reply:
For ubuntu 16, I am testing different approaches, as it seems we are exhausting the ram on the travis vm and end up in a timeout. Might be something to look into at github actions instead.